### ISSUE TITLE
Proxy become unavailable in k8s after running write operations in days

### ISSUE THREAD

**Issue by lucaspeng12138** [Alluxio/alluxio#18304]
**Alluxio Version:**
2.9.3

**Describe the bug**
Proxy become unavailable in k8s after running write operations in days, 39999 port is no longer working

**To Reproduce**
1. Deploy alluxio in k8s
2. Run write on proxy for one day long
3. Curl proxy port

**Expected behavior**
Proxy can work normally

**Urgency**
Import

**Are you planning to fix it**
No

**Additional context**

**Comment by humengyu2012** [comment]
Are there any logs in proxy?

**Comment by lucaspeng12138** [comment]
@humengyu2012  Yes, but only has INFO level infos. And only report metrics logs, no others.....

> Are there any logs in proxy?

**Comment by lucaspeng12138** [comment]
@humengyu2012 And find 100 CLOSE_WAIT connections on proxy network, and proxy has 
![image](https://github.com/Alluxio/alluxio/assets/100204617/946293b2-e1b2-423b-9ec5-250cadb49caa)

Following is jstack on proxy:
[jstack.log](https://github.com/Alluxio/alluxio/files/13162230/jstack.log)

**Comment by lucaspeng12138** [comment]
@humengyu2012 
yes，may be more than 256 concurrency write through proxy

**Comment by humengyu2012** [comment]
Earlier proxy versions might have memory leakage issues. Please check if this part of the code has been merged: https://github.com/Alluxio/alluxio/pull/17406.
